Source/diablo.cpp

```diff
@@ -327,17 +327,17 @@ void LeftMouseDown(int wParam)
 
 	if (MousePosition.y < PANEL_TOP || MousePosition.x < PANEL_LEFT || MousePosition.x >= PANEL_LEFT + PANEL_WIDTH) {
 		if (!gmenu_is_active() && !TryIconCurs()) {
-			if (QuestLogIsOpen && MousePosition.x > 32 && MousePosition.x < 288 && MousePosition.y > 32 && MousePosition.y < 308) {
+			if (QuestLogIsOpen && MousePosition.x > LEFT_PANEL_X + 32 && MousePosition.x < LEFT_PANEL_X + 288 && MousePosition.y > LEFT_PANEL_Y + 32 && MousePosition.y < LEFT_PANEL_Y + 308) {

```

Comment:
Stuff like this should end up some thing like this:

```cpp
for (auto panelType : UiPanels) {
    if (!IsPanelOpen(panelType)
        continue;
        
    Rectangle panel { GetPanelPosition(panelType), { 320, 352 } };
    if (panel.Contains(MousePosition)) {
        return true;
    }
}

return false;
```

Reply:
Got it.

This will become a function such like IsCursorOnPanel().